api/v2/sdks/sdk-ref-csharp-dotnet.md

```diff
@@ -45,53 +51,103 @@ The binary is published in this location:
 
 ## Initialization
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the EUID service. It also allows you to configure retry intervals in the event of errors.
+DSPs should create an instance of the `BidstreamClient` class.
+
+You will need to provide the values necessary for the SDK to authenticate with the EUID service.
+
+| Parameter | Description |
+| :--- | :--- |
+| `endpoint` | The endpoint for the EUID service. See [Environments](../getting-started/gs-environments). | 
+| `authKey` | The API key. See [EUID Credentials](../getting-started/gs-credentials). |
+| `secretKey` | The client secret. See [EUID Credentials](../getting-started/gs-credentials). |
+
+## Interface
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the EUID service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to EUID, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+The `BidstreamClient` class allows you to decrypt EUID tokens into raw EUIDs.
 
-## Interface 
+For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
 
-The interface allows you to decrypt EUID advertising tokens and return the corresponding raw EUID. 
+The `SharingClient` class allows you to encrypt raw EUIDs into EUID tokens and decrypt EUID tokens into raw EUIDs.
 
 >NOTE: When you use an SDK, you do not need to store or manage decryption keys.
 
-If you're a DSP, for bidding, call the interface to decrypt an EUID advertising token and return the EUID. For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
+### Encryption Response Content
```

Comment:
Remove the whole Encryption Response Content and Encryption Response Statuses section as they are for Sharing only

Reply:
Done, thx.